internal/system/agent/executor/operation_test.go

```diff
@@ -32,7 +32,7 @@ func TestOperationDoWithNoServices(t *testing.T) {
 
 	result := sut.Do([]string{}, "operationDoesNotMatter", []string{})
 
-	var emptyResult []interface{}
+	emptyResult := "incorrect or malformed body was passed in with the request"
```

Comment:
This string is defined in two separate places; consolidate it to a constant (if it remains after the other requested changes are implemented).

Reply:
So I had removed the definition `var emptyResult []interface{}` and replaced it by the definition emptyResult := "incorrect or malformed body was passed in with the request"`

Reply:
This string is defined in two separate places (inside both operation.go and operation_test.go); consolidate it to a constant (if it remains after the other requested changes are implemented).